src/js/tracing/addTracingExtensions.ts

```diff
@@ -1,97 +1,7 @@
-import type { Hub, Transaction } from '@sentry/core';
-import { addTracingExtensions, getCurrentHub, getMainCarrier } from '@sentry/core';
-import type { CustomSamplingContext, SpanContext, TransactionContext } from '@sentry/types';
-
-import { DEFAULT } from '../tracing/ops';
-import { ReactNativeTracing } from '../tracing/reactnativetracing';
-
 /**
  * Adds React Native's extensions. Needs to be called before any transactions are created.
  */
 export function _addTracingExtensions(): void {
-  addTracingExtensions();
-  const carrier = getMainCarrier();
-  if (carrier.__SENTRY__) {
-    carrier.__SENTRY__.extensions = carrier.__SENTRY__.extensions || {};
-    if (carrier.__SENTRY__.extensions.startTransaction) {
-      const originalStartTransaction = carrier.__SENTRY__.extensions.startTransaction as StartTransactionFunction;
-
-      /*
-        Overwrites the transaction start and finish to start and finish stall tracking.
-        Preferably instead of overwriting add a callback method for this in the Transaction itself.
-      */
-      const _startTransaction = _patchStartTransaction(originalStartTransaction);
-
-      carrier.__SENTRY__.extensions.startTransaction = _startTransaction;
-    }
-  }
+  // TODO: addTracingExtensions(); likely not needed in RN as it instruments global onerror and onunhandledrejections which are not use in RN
```

Comment:
true, this is not needed then! can be removed/dropped!

Reply:
removed